src/current/v24.2/api-support-policy.md

```diff
@@ -65,27 +65,48 @@ A stable API may be assigned a new version number in two situations:
+| [Cluster API][cluster-api]                       | Mixed    | Versioned independently from CockroachDB | For information on supported endpoints, versioning, and stability, refer to [Cluster API][cluster-api].                                                                                                                    | CockroachDB Advanced, CockroachDB Core                       |
+| [DB Console][db-console]                         | Unstable | N/A                                      | For stable access to the information present in this tool, use the [Cluster API][cluster-api].                                                                                                                             | CockroachDB Advanced, CockroachDB Core |
+| [Logging][logging-overview]                      | Mixed    | Versioned concurrently with CockroachDB  | Stability varies by [event type][eventlog]. Structured events are stable and unstructured events are unstable.                                                                                                             | CockroachDB Standard, CockroachDB Advanced, CockroachDB Core |
+| [ccloud CLI][ccloud-cli]                         | Mixed    | Versioned independently from CockroachDB | Default output is unstable. Specify the `–json` argument in the CLI for stable output that follows the versioning scheme.                                                                                                  | CockroachDB Cloud                                            |
+| [CockroachDB Cloud API][cloud-api]               | Stable   | Versioned independently from CockroachDB |                                                                                                                                                                                                                            | CockroachDB Cloud                                            |
+| [CockroachDB Cloud Console][cloud-console]       | Unstable | N/A                                      |                                                                                                                                                                                                                            | CockroachDB Cloud                                            |
+| [Advanced Debug endpoints][ui-debug-pages]       | Reserved | N/A                                      |                                                                                                                                                                                                                            | N/A                                                          |
+
+[pg-wire]: {% link {{ page.version.version }}/postgresql-compatibility.md %}
```

Comment:
Do you this change helps with maintenance of the file? I suspect that it may be a bit harder to troubleshoot than having the links directly in the table. Your call.

Reply:
I found it to be a worthy trade-off, making the table just narrow enough to be viewable and editable on my screen when zooming out